src/generators/web/template.html

```diff
@@ -6,7 +6,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="icon" href="https://nodejs.org/static/images/favicons/favicon.png"/>
   <title>{{title}}</title>
+  <meta name="description" content="Node.js® is a free, open-source, cross-platform JavaScript runtime environment that lets developers create servers, web apps, command line tools and scripts.">
```

Comment:
I'm open to suggestions for a better description

Reply:
So it's good 👍